addon/styles/_frost-button.scss

```diff
@@ -305,6 +297,54 @@ $frost-color-button-in-line-darken-2: darken($frost-color-blue-1, 15);
       box-shadow: none;
     }
   }
+  
+  // design = 'app-bar'
+  &.app-bar {
```

Comment:
Ok cool, I forgot that the intention of "design" was to avoid this type of debate over placement and make refactoring common parts easier - thanks for the reminder, carry on and ignore me